src/dispatch/case/flows.py

```diff
@@ -239,9 +239,35 @@ def case_triage_status_flow(case: Case, db_session=None):
 def case_escalated_status_flow(case: Case, organization_slug: OrganizationSlug, db_session=None):
     """Runs the case escalated transition flow."""
     # we set the escalated_at time
```

Comment:
Remove

Reply:
Oh wait, this and the other code below that was removed needs to stay and what was added to be removed as it already lives in the `case_to_incident_escalate_flow` function.